packages/app/src/components/Bookmarks/BookmarkFolderMenuItem.tsx

```diff
@@ -0,0 +1,233 @@
+                <FolderIcon isOpen={false} />
+                <span className="mx-2 ">{t('bookmark_folder.new_folder')}</span>
+              </DropdownItem>
+            )}
+
+            {hasChildren() && (<DropdownItem divider />)}
+
+            {item.children?.map(child => (
+              <div key={child._id} >
+                <div
+                  className='dropdown-item grw-bookmark-folder-menu-item'
+                  tabIndex={0} role="menuitem"
+                  onClick={e => onClickChildMenuItemHandler(e, child)}>
+                  <BookmarkFolderMenuItem
+                    onSelectedChild={() => setSelectedItem(null)}
+                    item={child}
+                    isSelected={selectedItem === child._id}
+                  />
+                </div>
+              </div>
+            ))}
+          </DropdownMenu>
+        )}
+      </>
+    );
+  }, [hasChildren,
+      isCreateAction,
+      isOpen, item.children,
+      onClickChildMenuItemHandler,
+      onClickNewBookmarkFolder,
+      onPressEnterHandlerForCreate,
+      t,
+      selectedItem,
+  ]);
+
+  return (
+    <>
+      <UncontrolledDropdown
+        direction="right"
+        className='d-flex justify-content-between '
+        isOpen={isOpen}
+        // toggle={toggleHandler}
+        onMouseLeave={onMouseLeaveHandler}
+      >
+        <div className='d-flex justify-content-start grw-bookmark-folder-menu-item-title'>
+          <input
+            type="radio"
+            checked={isSelected}
+            name="bookmark-folder-menu-item"
+            id={`bookmark-folder-menu-item-${item._id}`}
+            onChange={e => e.stopPropagation()}
+            onClick={e => e.stopPropagation()}
+          />
+          <label htmlFor={`bookmark-folder-menu-item-${item._id}`} className='p-2 m-0'>
+            {item.name}
+          </label>
+        </div>
+
+        <DropdownToggle
+          id={`bookmark-delete-button-${item._id}`}
+          className="text-danger ml-auto"
+          color="transparent"
+          onClick={e => onClickDeleteHandler(e)}
+        >
+          <i className="icon-fw icon-trash grw-page-control-dropdown-icon"></i>
+        </DropdownToggle>
+        <DropdownToggle
+          color="transparent"
+          onClick={e => e.stopPropagation()}
+          onMouseEnter={onMouseEnterHandler}
+        >
+          {hasChildren()
+            ? <TriangleIcon />
+            : (
+              <i className="icon-plus d-block pl-0" />
+            )}
+        </DropdownToggle>
+        {renderBookmarkSubMenuItem()}
+
+      </UncontrolledDropdown >
+      <UncontrolledTooltip
+        modifiers={{ preventOverflow: { boundariesElement: 'window' } }}
+        autohide={false}
+        placement="top"
+        target={`bookmark-delete-button-${item._id}`}
+        fade={false}
+      >
+        {t('bookmark_folder.delete')}
+      </UncontrolledTooltip>
+    </>
+  );
+};
+
+export default BookmarkFolderMenuItem;
```

Comment:
Please refactor to named export.
Don't forget to check deps for this component.